plugins/router/arris-sb6183

```diff
@@ -0,0 +1,267 @@
+            print("up_{0}.label Up Ch {1}".format(i, i))
+            print("up_{0}.type GAUGE".format(i))
+            # print("up_{0}.draw LINE1".format(i))
+
+        for i in range(1, DOWNCOUNT + 1):
+            name = "down_{0}".format(i)
+            print("\nmultigraph arris_power.{0}".format(name))
+            print("graph_title Downstream Power for Channel {0} (dBmV)".format(i))
+            print("graph_category network")
+            print("power.label dBmV")
+            print("power.type GAUGE")
+            print("power.draw LINE1")
+        for i in range(1, UPCOUNT + 1):
+            name = "up_{0}".format(i)
+            print("\nmultigraph arris_power.{0}".format(name))
+            print("graph_title Upstream Power for Channel {0} (dBmV)".format(i))
+            print("graph_category network")
+            print("power.label dBmV")
+            print("power.type GAUGE")
+            print("power.draw LINE1")
+
+        # SNR
+        print("\nmultigraph arris_snr")
+        print("graph_title Arris Signal-to-Noise Ratio (dB)")
+        print("graph_vlabel SNR (dB)")
+        print("graph_category network")
+        for i in range(1, DOWNCOUNT + 1):
+            print("down_{0}.label Ch {1}".format(i, i))
+            print("down_{0}.type GAUGE".format(i))
+            print("down_{0}.draw LINE1".format(i))
+
+        for i in range(1, DOWNCOUNT + 1):
+            name = "down_{0}".format(i)
+            print("\nmultigraph arris_snr.{0}".format(name))
+            print("graph_title SNR on Channel {0} (dB)".format(i))
+            print("graph_vlabel SNR (dB)")
+            print("graph_category network")
+            print("snr.label dB")
+            print("snr.type GAUGE")
+            print("snr.draw LINE1")
+
+        # ERRORS
+        print("\nmultigraph arris_error")
+        print("graph_title Arris Channel Errors")
+        print("graph_category network")
+        print("graph_args --base 1000")
+        print("graph_vlabel errors/sec")
+        print("graph_category network")
+        print("corr.label Corrected")
+        print("corr.type DERIVE")
+        print("corr.min 0")
+        print("uncr.label Uncorrectable")
+        print("uncr.type DERIVE")
+        print("uncr.min 0")
+        print("uncr.warning 1")
+
+        for i in range(1, DOWNCOUNT + 1):
+            name = "down_{0}".format(i)
+            print("\nmultigraph arris_error.{0}".format(name))
+            print("graph_title Channel {0} Errors".format(i))
+            print("graph_args --base 1000")
+            print("graph_vlabel errors/sec")
+            print("graph_category network")
+            print("corr.label Correctable")
+            print("corr.type DERIVE")
+            print("corr.min 0")
+            print("uncr.label Uncorrectable")
+            print("uncr.type DERIVE")
+            print("uncr.min 0")
+            print("uncr.warning 1")
+
+        sys.exit(0)
+
+    if sys.argv[1] == "autoconfig":
+        try:
```

Comment:
Here you would also need to verify the availability of the non-core modules (`lxml` and `requests`).